tests/devices/test_default_qubit_tf.py

```diff
@@ -114,13 +114,27 @@ def init_state(scope="session"):
 
     def _init_state(n):
         """random initial state"""
+        np.random.seed(4214152)

```

Comment:
I am surprised we didn't set the seed in the past 😮.

Reply:
Yeah, I was, too :sweat_smile: